### PR TITLE
Creat project

### PR DESCRIPTION
**Pull request by lesang458**

### CODE REVIEW

File: assignment-project/e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { AppPage } from './app.po';
+import { browser, logging } from 'protractor';
+
+describe('workspace-project App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
```
Comment (tranmanhlinh) [inline assignment-project/e2e/src/app.e2e-spec.ts:9]: Remove al test files.

File: assignment-project/src/app/modules/book/book.module.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
```
Comment (tranmanhlinh) [inline assignment-project/src/app/modules/book/book.module.ts:4]: Keep only one empty line.

File: assignment-project/src/app/modules/category/category.module.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
```
Comment (tranmanhlinh) [inline assignment-project/src/app/modules/category/category.module.ts:4]: Same here.

File: assignment-project/src/app/modules/user/user.module.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+
```
Comment (tranmanhlinh) [inline assignment-project/src/app/modules/user/user.module.ts:5]: Same here.

File: assignment-project/e2e/src/app.e2e-spec.ts (hunk 1 above)
Comment (lesang458) [reply assignment-project/e2e/src/app.e2e-spec.ts]: done

File: assignment-project/src/app/modules/book/book.module.ts (hunk 2 above)
Comment (lesang458) [reply assignment-project/src/app/modules/book/book.module.ts]: done

File: assignment-project/src/app/modules/category/category.module.ts (hunk 3 above)
Comment (lesang458) [reply assignment-project/src/app/modules/category/category.module.ts]: done

File: assignment-project/src/app/modules/user/user.module.ts (hunk 4 above)
Comment (lesang458) [reply assignment-project/src/app/modules/user/user.module.ts]: done

**Pull request merged**